src/eradius_proxy.erl

```diff
@@ -36,9 +36,10 @@
 
 -type route() :: eradius_client:nas_address().
 -type routes() :: [{Name :: string(), route()}].
+-type undefined_route() :: {undefined, 0, []}.
 
 radius_request(Request, _NasProp, Args) ->
-    DefaultRoute = proplists:get_value(default_route, Args),
+    DefaultRoute = proplists:get_value(default_route, Args, {undefined, 0, []}),
```

Comment:
you probably don't need this change if you have `send_to_server` which able to process `undefined` as the second argument.

Reply:
right